nodejs/sample-apps/aws-sdk/deploy/wrapper/main.tf

```diff
@@ -1,6 +1,13 @@
+locals {
+  collector_layer_arn = "arn:aws:lambda:${data.aws_region.current.name}:${var.account_id}:layer:opentelemetry-collector-arm64-0_15_0:1"
```

Comment:
It might be good to have layer versions (with default values of current latest versions) as variable defined in the `variables.tf` like AWS account id.